### ISSUE TITLE
NTR: Hyaluronic acid high in urine (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8834]
New term request Hyaluronic acid high in urine (elthat_100717235535):   UMLS: 4015469

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000391
- label: Elevated urine hyaluronic acid level
- parents: [Urinary glycosaminoglycan excretion (HP:0003541)](https://hpo.jax.org/app/browse/term/HP:0003541)
- pmid: 
- def: Excessive amount of hyaluronic acid in urine.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8834
- synonyms: Hyaluronic acid high in urine(NONE)
